document/js-api/index.bs

```diff
@@ -1445,12 +1445,12 @@ interface Suspending {
     1. Otherwise,
         1. Assert: |result| is a [=list=] of WebAssembly values.
         1. Let |outArity| be the [=list/size=] of |result|.
-        1. If |outArity| is 0, return undefined.
+        1. If |outArity| is 0, let |jsReturnValue| be undefined.
```

Comment:
Should we factor out this section with the same spec steps found in "call an Exported Function"?